### PR TITLE
Update Gazebo version references in Linux install pages

### PR DESCRIPTION
**Pull request by j-rivero**
Give the install documents a good update on versions available. Simplify the Ubuntu page by removing outdated warnings and info.

### CODE REVIEW

File: install_other_linux/tutorial.md [hunk 1]
```diff
@@ -8,30 +8,28 @@ Debian, Fedora, Arch and Gentoo.
...
-version, the version of Gazebo available is different:
+as an official package.
 
- * Rawhide is usually hosting one of the latest releases
- * Fedora 31: gazebo-9.x
- * Fedora 30: gazebo-9.x
- * Fedora 29: gazebo-8.x
+ * Rawhide: gazebo-10
+ * Fedora 35: gazebo-10.x
+ * Fedora 36: gazebo-10.x
```
Comment (chapulina) [inline install_other_linux/tutorial.md:32]: Gazebo 10 is EOL. If there's no Gazebo 9 or 11 on these platforms, should we just remove them?

File: install_ubuntu/tutorial_11.0.md [hunk 2]
```diff
@@ -61,35 +61,6 @@ installation](http://gazebosim.org/tutorials?tut=ros_wrapper_versions&cat=connec
...
-    For example, installing gazebo11 on Ubuntu Xenial (16.04) will produce the error above.
-    Hint: Take a look at "Project Status" section at [http://gazebosim.org/#status](http://gazebosim.org/#status),
-    next to each version is the supported ubuntu versions and ROS versions.
-
-
 1. Check your installation
 
         gazebo
-
-## Gazebo in different deb packages
```
Comment (chapulina) [inline install_ubuntu/tutorial_11.0.md:82]: It's not clear to me what's wrong with this section

**Review by chapulina — APPROVED** [review]

File: install_other_linux/tutorial.md (hunk 1 above)
Comment (j-rivero) [reply install_other_linux/tutorial.md:32]: Although Gazebo10 is EOL by our side (upstream) the distributions are free to keep shipping unsupported versions until they have a better alternative (i.e: In Debian we still have ogre-1.9).

I think it does not hurt to mention that here that Fedora is in version 10, maybe could help people to see that there is a need to update and someone contribute to Fedora to update it.

File: install_ubuntu/tutorial_11.0.md (hunk 2 above)
Comment (j-rivero) [reply install_ubuntu/tutorial_11.0.md:82]: Nothing was bad in the section, it was created when we first split up the gazebo monolithic package to help people understanding what's going on. That happened years ago, I found no reason to keep the section here and found that people can be confused with it. Just a personal opinion.

**Pull request merged**